portality/models/v2/journal.py

```diff
@@ -70,8 +70,8 @@ def find_by_issn(cls, issns, in_doaj=None, max=10):
         return records
 
     @classmethod
-    def issns_by_owner(cls, owner):
-        q = IssnQuery(owner)
+    def issns_by_owner(cls, owner, in_doaj=None):
```

Comment:
The default to `None` here is at odds with the default to `False` of its usage with the `ISSNQuery`